src/website/app/demos/Card/index.js

```diff
@@ -51,6 +52,7 @@ Cards represent a gateway to more detailed information in another app view.`
     doc: cardBlockDoc,
     examples: cardBlockExamples,
     slug: 'card-block',
+    syntax: `import { CardBlock } from 'mineral-ui/Card'`,
```

Comment:
I wonder if subcomponents should be `import Card, { CardBlock } from 'mineral-ui/Card'`, since they're not useful outside of the default exported component? Just a thought.

Reply:
👍